src/hotspot/os_cpu/windows_x86/os_windows_x86.cpp

```diff
@@ -416,41 +416,41 @@ void os::print_context(outputStream *st, const void *context) {
 
   st->print_cr("Registers:");
 #ifdef AMD64
-  st->print(  "RAX=" INTPTR_FORMAT, uc->Rax);
-  st->print(", RBX=" INTPTR_FORMAT, uc->Rbx);
-  st->print(", RCX=" INTPTR_FORMAT, uc->Rcx);
-  st->print(", RDX=" INTPTR_FORMAT, uc->Rdx);
+  st->print(  "RAX=0x%016llx", uc->Rax);
+  st->print(", RBX=0x%016llx", uc->Rbx);
+  st->print(", RCX=0x%016llx", uc->Rcx);
+  st->print(", RDX=0x%016llx", uc->Rdx);
```

Comment:
Why?

Reply:
All of these are either DWORD64 or DWORD, none are actually intptr_t's or uintptr_t's

Reply:
In that case UINT64_FORMAT_X would be more correct. But again, unless there is an important reason to do so, I would leave changing this to when someone changes/moves around the code.